types.d.ts

```diff
@@ -1,24 +1,41 @@
 import { bits } from 'discord-rose/dist/utils/Permissions'
 import { CommandContext } from "discord-rose/dist/structures/CommandContext";
 
+type bitKey = (keyof typeof bits)[]
+
 declare module 'discord-rose/dist/typings/lib' {
   interface CommandOptions {
     /**
      * Whether or not the executing user has said permissions
      */
-    userPerms?: (keyof typeof bits)[]
+    userPerms?: bitKey
     /**
      * Whether or not the bot has said permissions
      */
-    myPerms?: (keyof typeof bits)[]
+    myPerms?: bitKey
   }
 }
 
-type msgFunction = (ctx: CommandContext) => string
+type humanReadableBits = {
+  [key in bitKey[number]]?: string
+}
+
+type msgFunction = (perms: (keyof typeof bits)[], ctx: CommandContext) => string
```

Comment:
same thing here, and by the way no don't just change the order in the code, ctx has to come first otherwise it'd be a breaking change.